logic/src/commonMain/kotlin/com/wire/kalium/logic/CoreLogic.kt

```diff
@@ -46,6 +46,8 @@ abstract class CoreLogicCommon(
     @Suppress("MemberVisibilityCanBePrivate") // Can be used by other targets like iOS and JS
     abstract fun getSessionScope(userId: UserId): UserSessionScope
 
+    abstract fun deleteSessionScope(userId: UserId) // TODO remove when proper use case is ready
```

Comment:
i will replace it with a stand alone usecase once my PR is merged

Reply:
Please make sure we call `sessionScope.cancel()` too!
This way we force some of our "lifecycle" tasks to stop too, like CallManager and some Sync stuff.

Reply:
My solution is temporary, as Mo said he works on a proper use case, but I added `.cancel()` to this temporary code